pagopa/pn-frontend
# fix: fix unmet peer dependencies

## Short description
This pr fixes every unmet peer dependency warning except react-scripts@^4.0.0 required by craco@6.4.5.
I tried to update craco to it's latest version (7.0.0) but this caused the issue reported [here](https://github.com/yarnpkg/yarn/issues/7807)

## List of changes proposed in this pull request
- add unmet peer dependencies as dependency or devDependency to fix warnings thrown by yarn install

## How to test
Run yarn refresh:monorepo and verify the warnings about unmet peer dependency has reduced to only the one about react-scripts@^4.0.0
